api/src/main/java/io/cloudevents/CloudEventAttributes.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2018-Present The CloudEvents Authors
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.cloudevents;
+
+import io.cloudevents.lang.Nullable;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.net.URI;
+import java.time.ZonedDateTime;
+import java.util.Set;
+
+/**
+ * The marker interface for CloudEvents attributes
+ *
+ * @author fabiojose
```

Comment:
What is the policy around author tags. I see we have some lingering names but clearly fabio did not author this class. Just curious

Reply:
not sure we have a policy about tags, for now i just remove it since it's clearly incorrect